### DISCUSSION TITLE
Monocle layout

### DISCUSSION THREAD

**Discussion by Okanda** [YaLTeR/niri#398] [Ideas]
Hello. 

In BSPWM there is a layout called monocle. Windows are maximized but without any gaps. 
This is pretty cool when we work for a long time on a single window (web, Writer, photo treatment,. . . ). 

Is this can be difficult to implement? 

I absolutely don't have any skills in programmation. I just ask. .. in case ;)

Thank you.

**Comment by YaLTeR** [comment]
You could set gaps to 0 then maximize a column?

**Reply by Okanda** [reply]
It could be cool (well, for me) to have the possibility to activate it just when I want with a shortcut.

**Reply by YaLTeR** [reply]
You could run a script that programmatically changes the line in the config.

**Reply by Okanda** [reply]
I will try. thx
